.github/workflows/main.yml

```diff
@@ -31,6 +32,11 @@ jobs:
 
       - name: Install dependencies
         run: composer install
+        if: matrix.php-version != '8.0'
+
+      - name: Update dependencies
+        run: rm composer.lock && composer update
+        if: matrix.php-version == '8.0'
```

Comment:
As mentioned in https://github.com/Ecodev/graphql-upload/pull/14#issuecomment-735770619 , the dependency on php-cs-fixer also needs to be removed, e.g.
`composer remove --dev --no-update friendsofphp/php-cs-fixer`

Hopefully it works then 🤞